Formula/ripasso-cursive.rb

```diff
@@ -0,0 +1,30 @@
+class RipassoCursive < Formula
+  desc "Tui password manager compatible with 'pass', written in rust"
+  homepage "https://github.com/cortex/ripasso"
+  url "https://github.com/cortex/ripasso/archive/refs/tags/release-0.6.3.tar.gz"
+  sha256 "2f43e48c3e1064bef47c89269e4797f216b7600b850f3857cfce7e9716ef8f7d"
+  license "GPL-3.0-only"
+
+  depends_on "libxcb" => :build
+  depends_on "llvm" => :build
+  depends_on "pkg-config" => :build
+  depends_on "rust" => :build
+  depends_on "gpgme"
+  depends_on "libgpg-error"
+  depends_on "nettle"
+
+  def install
+    ENV["SHELL_COMPLETIONS_DIR"] = buildpath
+    my_cargo_args = std_cargo_args.map(&:clone)
+    my_cargo_args.pop
+    my_cargo_args.push("cursive")
+    system "cargo", "install", "ripasso-cursive", *my_cargo_args
+
+    man1.install "target/man-page/cursive/ripasso-cursive.1"
+  end
+
+  test do
+    output = shell_output("#{bin}/ripasso-cursive -h")
+    assert_match "password manager", output
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests.

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
I think the best might be to run the unit test suite of the program, `cargo test -p ripasso -p ripasso-cursive` but that is maybe in the wrong phase? Is the test phase more of a smoke test that the program binary behaves?

Reply:
It's exactly that. All we want to know is if no Homebrew logic went wrong. It's up to upstream to write the code correctly.

Reply:
Since the program compiles to a single executable there isn't that many things that could go wrong, as long as the build completes without errors.

And the whole program is with a curses interface so hard to do a smoke test of. I'll try to figure out a way to do a better test.

Reply:
I reworked the test by starting the program and capturing the output with the help of tmux, this adds a test dependency, but I guess that isn't a big deal?